hack/kind-up.sh

```diff
@@ -52,6 +53,10 @@ parse_flags() {
     --multi-zonal)
       MULTI_ZONAL=true
       ;;
+    --with-lpp-resize-support)
+        shift
+        WITH_LPP_RESIZE_SUPPORT="${1}"
+        ;;
```

Comment:
Nit, for consistency reason:

```suggestion
      shift
      WITH_LPP_RESIZE_SUPPORT="${1}"
      ;;
```

Reply:
Fixed, thanks.